### DISCUSSION TITLE
GitHub achievements

### DISCUSSION THREAD

**Discussion by kimsh153** [community/community#18775] [Other Feature Feedback, Questions, & Ideas]
how to get
![](https://cdn.discordapp.com/attachments/820873397113585674/987193079612211210/2022-06-17_12.12.48.png)
this badge?

**Comment by baekteun** [comment]
Contributed code to repositories used in the [Mars 2020 Helicopter Mission](https://github.com/readme/nasa-ingenuity-helicopter)

**Comment by Ty-Irving** [comment]
https://github.com/Schweinepriester/github-profile-achievements

**Comment by thomscoder** [comment]
You can't get that badge anymore. I've wrote [a quick overview](https://nine-five.ghost.io/github-achievements/) about Achievements.

**Comment by kangketikonlen** [comment]
How about next achievement?

**Comment by haseeb-xd** [comment]
Here you can see all achievements : https://github.com/Schweinepriester/github-profile-achievements/

**Comment by melvincwng** [comment]
@kimsh153, the Mars helicopter achievement badge cannot be obtained as of today. It was only available for developers to obtain back in 2020. Hope this clarifies.

**Comment by JonasJs** [comment]
Check out this repository, it has everything about all achievements in detail 😄

https://github.com/Schweinepriester/github-profile-achievements

**Comment by skmn3** [comment]
I have contributed to this repository.
https://github.com/Schweinepriester/github-profile-achievements

**Comment by Do-hyun-Kim** [comment]
Check this repository and it should solve your problem.
https://github.com/Schweinepriester/github-profile-achievements
